asyncapi/asb/Dependencies.toml

```
@@ -6,6 +6,16 @@
 [ballerina]
 dependencies-toml-version = "2"
 
+[[package]]
+org = "ballerina"
+name = "io"
+version = "1.3.1"
+scope = "testOnly"
```

```suggestion
scope = "testOnly"
```
This comes on `bal test` (I guess). Can you remove this `dependencies.toml` and `bal pack` again.